.github/workflows/bokeh-ci.yml

```diff
@@ -191,6 +192,7 @@ jobs:
           verbose: true
 
   unit-test:
+    if: "startsWith(github.event.path, 'src/bokeh/')"
```

Comment:
Unfortunately this is not sufficient, at least as things currently stand. Some of the unit-tests, e.g. for PNG and SVG generation, utilize selenium to actually generate outputs to compare. (I think there are a few other places as well) So then this would need to depend on `bokehjs` as well. 

We could potentially consider anything involving selenium to be an end-to-end "integration" test, but we would need to figure out what to do with the current selenium integration tests, which are disabled due to flakiness. Keeping tests that verify actual, correct PNG/SVG output *starting from python* is definitely a non-negotiable for me, wherever those tests end up.

Reply:
@DerTimonius I think if you add `bokehjs` to unit tests and also update for the notes https://github.com/bokeh/bokeh/pull/13046#issuecomment-1494651982 then this could be merged for now, with other improvements (e.g. splitting off the selenium tests out of unit tests) happening later.